src/proto/message_translation/power_frame_msg.hpp

```diff
@@ -112,6 +112,19 @@ TbotsProto_PowerStatus inline createNanoPbPowerStatus(
     return status;
 }
 
+/**
+ * Calculates width of pulse for chicker
+ * @param kick_constant
+ * @param kick_coefficient
+ * @param speed Speed of the desired kick in m/s
+ * @return Width of pulse
+ */
+uint32_t calculateChickerPulseWidth(int kick_constant, double kick_coefficient,
```

Comment:
You'll need to mark this function as `inline`. Functions defined in header files need to be marked as `inline` to avoid violating the [One Definition Rule](https://en.wikipedia.org/wiki/One_Definition_Rule)